source/loader/ze_loader_internal.h

```diff
@@ -26,6 +26,18 @@
 #include "spdlog/spdlog.h"
 namespace loader
 {
+    ///////////////////////////////////////////////////////////////////////////////
```

Comment:
Optional, put a comment here that the ordering here is critical and/or used by sorting algorithms, so care should be used when adding.  Also this should NOT be used anyplace else so we can modify this ENUM  (I see this is internal so should be fine, but let's ensure it stays internal) without ABI issues.

Reply:
This remains internal and is not used outside of the loader nor released. I updated the comment accordingly.